user_mtr_on_j1.h

```diff
@@ -229,7 +229,7 @@ extern "C" {
 //! \brief Uncomment the motor which should be included at compile
 //! \brief These motor ID settings and motor parameters are then available to be used by the control system
 //! \brief Once your ideal settings and parameters are identified update the motor section here so it is available in the binary code
-#define USER_MOTOR TMotor_Antigravity_MN7005_115kv
```

Comment:
Small comment: Maybe leave the old `#define` for `TMotor_Antigravity_MN7005_115kv` here and just comment it out? This way, if someone is using the motor it's easier to enable it again.

Reply:
done